lib/ansible/modules/network/nxos/nxos_interface.py

```diff
@@ -221,21 +221,22 @@ def get_manual_interface_attributes(interface, module):
     if get_interface_type(interface) == 'svi':
         command = 'show interface {0}'.format(interface)
         try:
-            body = execute_show_command(command, module)[0]
+            body = run_commands(module, [command])[0]
         except IndexError:
             return None
 
-        command_list = body.split('\n')
-        desc = None
-        admin_state = 'up'
-        for each in command_list:
-            if 'Description:' in each:
-                line = each.split('Description:')
-                desc = line[1].strip().split('MTU')[0].strip()
-            elif 'Administratively down' in each:
-                admin_state = 'down'
-
-        return dict(description=desc, admin_state=admin_state)
+        if body and module.params['transport'] == 'cli':
```

Comment:
So we want to do the same for nxapi. Tested it with nxapi mode. I am not going to restrict it for cli mode.
Modified the PR.